app/services/api/v3/participant_declarations_query.rb

```diff
@@ -36,13 +36,8 @@ def participant_declarations_for_pagination
 
       def participant_declarations_from(paginated_join)
         sub_query = ParticipantDeclaration
-          .select("participant_declarations.*", "COALESCE(jsonb_agg(DISTINCT participant_identities_mentor_profiles.user_id) FILTER (WHERE participant_identities_mentor_profiles.user_id IS NOT NULL), '[]') AS transient_mentor_user_id")
+          .select("participant_declarations.*")
           .joins("INNER JOIN (#{paginated_join.to_sql}) as tmp on tmp.id = participant_declarations.id")
```

Comment:
not sure we need the subquery anymore as we no longer need to group by participant declaration id, maybe we can move the inner join into one query with the one below?